dashboard/visualization/plots.py

```diff
@@ -61,18 +61,23 @@ def plot_projection_2d(
     return fig
 
 
-def visualize_multiple_plates(df: pd.DataFrame, plate_array: np.ndarray) -> go.Figure:
+def visualize_multiple_plates(
+    df: pd.DataFrame, plate_array: np.ndarray, rows: int = 3, cols: int = 3
+) -> go.Figure:
     """
     Visualize plate values on subplots 3x3
 
     :param df: DataFrame with plates
     :param plate_array: array with plate values
```

Comment:
How does the figure look like when the number of plates is different from `rows`x`cols`?

I'd be nice if we could handle it so that e.g. if we have 8 plates but still want 3x3 grid, the function returns the plot like:

x x x
x x x
x x

where 'x' represents a plate subplot

Reply:
when the number of plates is lower it works like that:
![image](https://github.com/zuzg/drug-screening/assets/81712088/b90df602-bac9-4e9e-8680-dca202171c4d)
when the number is greater it just plots first x plates